lib/boilerPlate.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const util = require('util');
+
+/**
+ * Error for whatever you want!
+ *
+ * @extends Error
+ * @class Errors.CoolStuff
+ * */
```

Comment:
Given your class name I think CoolStuff would not work, and does not give the user an example that it should match the name of the class.

This should be .BoilerPlate right? Otherwise the person extending doesn't know it matches the class name. They think they can just put some cool stuff there,v not that it is a disciplined name matching the class name.